django_app/frontend/style.scss

```diff
@@ -1,4 +1,33 @@
 $govuk-assets-path: "govuk-assets/";
 $govuk-font-family: sans-serif;
+$iai-pink: #b62777;
 
-@import "django_app/frontend/node_modules/govuk-frontend/dist/govuk/all";
+@import "../node_modules/govuk-frontend/dist/govuk/all";
```

Comment:
I think this is causing tests to fail - the previous version is safer

Reply:
Oh, I didn't mean to change that - I think that's a mistake when resolving conflicts. I'll sort now - thanks!

Reply:
I'm getting this error when changing. Do you get the same?
Error: File to import not found or unreadable: django_app/frontend/node_modules/govuk-frontend/dist/govuk/all